Makd.mak

```diff
@@ -267,6 +270,9 @@ PKGVERSION := $(shell echo $(VERSION) | \
 # http://www.gnu.org/software/make/manual/make.html#Text-Functions
 TEST_FILTER_OUT := $C/$(SRC)/%/main.d
 
+# Directories to search for files to convert using d1to2fix
+D1TO2FIX_DIRS := $C
```

Comment:
There is a bit of inconsistency in the use of `:=` vs `?=` I was just following the variables close by. `:=` is a bit misleading though, because if you pass a variable via command-line for example, it will override variables defined with `:=` anyway. `?=` is only for variables defined inside the makefile and since the `Build.mak` is processed after this was defined, you can still override it.

It won't be definable via `Config.mak`, which is where I think the things got a bit more weird. But since this is a pre-existing problem, I just want to be consistent with the other similar variables (like the `TEST_FILTER_OUT` above) for now, to avoid further confusion.

At some point we should review how variables are defined and try to be more consistent and conscious about what should be defined in `Config.mak` and what in `Build.mak`. Is a hard job with little gain, though, and I guess that's why it was never addressed.